### PR TITLE
Remove kargakis from OWNERS, add tnozicka

### PR DESCRIPTION
**Pull request by 0xmichalis**
/assign @smarterclayton 
/cc @janetkuo @tnozicka

### CODE REVIEW

**Comment by kargakis** [comment]
@smarterclayton needs an approval

**Review by janetkuo — APPROVED** [review]

**Review by janetkuo — CHANGES_REQUESTED** [review]
@kargakis would you add tnozicka as reviewer but not as approver?

**Comment by kargakis** [comment]
@janetkuo @tnozicka is added as a reviewer already. Did you mean that I should add him as an approver too?

**Comment by janetkuo** [comment]
I meant reviewer only, but not approver

**Comment by kargakis** [comment]
@janetkuo I already do that..

File: test/integration/deployment/OWNERS
Comment (janetkuo) [inline test/integration/deployment/OWNERS]: remove this line

Comment (kargakis) [reply test/integration/deployment/OWNERS]: done

**Review by janetkuo — APPROVED** [review]

**Comment by kargakis** [comment]
@smarterclayton needs an approval

**Pull request merged**
